src/g2_gbytesc.f

```diff
@@ -1,57 +1,82 @@
+!>    @file
```

Comment:
@GeorgeGayno-NOAA am I correct in rembering that doxygen wants us to use C> for fixed format code? (That is, F77 code such as this?)

I seem to recall we discovered that in the UFS_UTILS doxygenation...

Reply:
If so, I can change it back to C>.

Reply:
Let's wait for George's comments, and go further on this one.

Reply:
IIRC (and I may not) this manifested in doxygen warnings that did not go away until I changes the comments back to C>...

You can check the doxygen warnings in the CI build, and see if there are any warnings for this code. All warnings must be eliminated, so we can turn on automatic checking of documentation completeness in the CI system.

Reply:
https://hang-lei-noaa.github.io/NCEPLIBS/NCEPLIBS-g2/html/g2__gbytesc_8f.html
This has been correctly shown on gh_pages

Reply:
@edwardhartnett The UFS_UTILS sfcsub.F routine is F77. To get it to work, I used the C> syntax at the top, but the !> syntax through the remainder of the file.  

See:  https://github.com/NOAA-EMC/UFS_UTILS/blob/develop/sorc/global_cycle.fd/sfcsub.F